lib/transaction_checker.go

```diff
@@ -28,12 +28,7 @@ func CheckTransactionSource(c sebakcommon.Checker, args ...interface{}) (err err
 }
 
 func CheckTransactionCheckpoint(c sebakcommon.Checker, args ...interface{}) (err error) {
-	checker := c.(*TransactionChecker)
-	if _, err = sebakcommon.ParseCheckpoint(checker.Transaction.B.Checkpoint); err != nil {
-		err = sebakerror.ErrorTransactionInvalidCheckpoint
-		return
-	}
-
+	//checker := c.(*TransactionChecker)
```

Comment:
I'm not sure if that's correct. Of course `ParseCheckpoint` needs to be removed, but it feels like this is missing a check ?